doc/source/conf.py

```diff
@@ -155,19 +155,27 @@
     },
     "header_links_before_dropdown": 6,
     "icon_links": [
+        {
+            "name": "GitHub",
+            "url": "https://github.com/scikit-image/scikit-image",
+            "icon": "fa-brands fa-github",
+        },
         {
             "name": "PyPI",
             "url": "https://pypi.org/project/scikit-image/",
             "icon": "fa-solid fa-box",
         },
     ],
+    "navbar_align": "left",
```

Comment:
This and the added / removed configuration in "icon_links" and "github_url" accounts for some updates between 0.13 and 0.14.